app/src/main/java/com/neudesic/myapplication/ui/screens/UserListScreen.kt

```diff
@@ -0,0 +1,35 @@
+package com.neudesic.myapplication.ui.screens
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.sp
+import com.neudesic.myapplication.R
+
+@Composable
+fun UserListScreen() {
```

Comment:
I am useless for compose since I don't really work on UI. However, Twitter recently posted up some pretty good rules for compose development. Nonetheless, this looks good. 
https://twitter.github.io/compose-rules/rules/